src/common/LogEntry.h

```diff
@@ -15,7 +15,11 @@
 #ifndef CEPH_LOGENTRY_H
 #define CEPH_LOGENTRY_H
 
+#include <fmt/core.h>
```

Comment:
why? `fmt/core.h` is already included by, i think, all of the `fmt/*.h` headers? or do you want to explicitly include the very header which has the `FMT_VERSION` macro definition?

Reply:
This was a "copy the documentation thing", happy to remove it

Reply:
c3e05b5732af6a0722591f59b991178de3042e45